src/java/org/apache/cassandra/config/Config.java

```diff
@@ -1296,5 +1296,6 @@ public static void log(Config config)
 
     public volatile DurationSpec.LongMillisecondsBound progress_barrier_timeout = new DurationSpec.LongMillisecondsBound("3600000ms");
     public volatile DurationSpec.LongMillisecondsBound progress_barrier_backoff = new DurationSpec.LongMillisecondsBound("1000ms");
+    public volatile DurationSpec.LongSecondsBound discovery_timeout = new DurationSpec.LongSecondsBound("30s");
```

Comment:
Should we make this way higher by default, like 10 or even 30 minutes? I'm just concenred that it is barely enough for a local sizeable cluster to start. Maybe we should make it higher.